test/basic.test.js

```diff
@@ -359,8 +359,8 @@ test('set the base to null and use a serializer', async ({ is, same }) => {
   const stream = sink()
   const instance = pino({
     base: null,
-    serializers: {
-      [Symbol.for('pino.*')]: (input) => {
+    formatters: {
+      log (input) {
```

Comment:
This test says "use a serializer". I think we need to update the test description